app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -21,7 +22,7 @@
   error_id = "error-#{SecureRandom.hex(4)}"
 
   css_classes = %w(gem-c-input govuk-input)
-  css_classes << "govuk-input--error" if has_error
+  css_classes << "govuk-input--error" if has_error || error_style
```

Comment:
It looks like we can already override the `has_error` attribute on the input component, so do we need the extra attribute? Minor, but if we keep it, I was expecting it to contain some classes or something :-|.

Reply:
How can we override the `has_error`? happy to change the name with a better proposal.

Reply:
I noticed it's an undocumented parameter of the input component :-).

Reply:
Yes, but isn't it overwritten on [L20](https://github.com/alphagov/govuk_publishing_components/blob/337e3f02ff232fc3d2682438932826388a863d34/app/views/govuk_publishing_components/components/_input.html.erb#L20)?

Reply:
Oh yeah, sorry. But instead of adding a new param, you could just add a `||`, since `has_error` is a pretty good name :-).

Reply:
ok, will re-use it then.

Reply:
Gave it a try, but we're using `has_error` for more than styling an input (which is what we want in this case); we use it to generate the red border on form group and attach an ID to the aria describedby attribute. So not really an option to recycle it.